src/agents/atomdb_broker/AtomDBProxy.cc

```diff
@@ -161,6 +183,20 @@ void AtomDBProxy::handle_add_atoms(const vector<string>& tokens) {
     }
 }
 
+void AtomDBProxy::handle_delete_atoms(const vector<string>& args) {
```

Comment:
The name `handle_delete_atoms()` would be OK if we didn't use the term `handle` so intensively all along the code meaning something totally different. I suggest changing it. Perhaps using the terms "callback" or "piggyback". BTW I understand you just kept the naming convention which was already in the class. So perhaps we should change `handle_add_atoms()` as well.